lib/dartdoc.dart

```diff
@@ -279,13 +286,40 @@ class DartDoc {
     // Use the includeExternals.
     for (Source source in context.librarySources) {
       LibraryElement library = context.computeLibraryElement(source);
+      if (library == null) {
```

Comment:
Are you seeing use cases when the analyzer returns null for LibraryElement . Pretty sure it should not do so. Could you check with @bwilkerson on this?